### PR TITLE
Changing the cron trigger to hopefully reduce the change of jobs being dropped.

### PR DESCRIPTION
**Pull request by jamesemery**

### CODE REVIEW

File: .github/workflows/gatk-nightly-docker.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 name: Build and Upload GATK Nightly Docker Image
 on:
   schedule:
-    # trigger the job the image at 1am est every night (UTC 20:00)
-    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
-    - cron: '0 20 * * *'
+    - cron: '17 20 * * *' # trigger the job the image at 1:17am est every night (UTC 20:17)
```
Comment (droazen) [inline .github/workflows/gatk-nightly-docker.yml:4]: UTC 20:17 is actually 4:17 PM Eastern, I believe...

**Review by droazen — CHANGES_REQUESTED** [review]

File: .github/workflows/gatk-nightly-docker.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
 name: Build and Upload GATK Nightly Docker Image
 on:
   schedule:
-    # trigger the job the image at 1am est every night (UTC 20:00)
-    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
-    - cron: '0 20 * * *'
+    - cron: '17 05 * * *' # trigger the job the image at 1:17am est every night (UTC 05:17)
```
Comment (droazen) [inline .github/workflows/gatk-nightly-docker.yml:4]: It's actually 1:17 AM EDT (Eastern Daylight Time), but 12:17 AM EST (Eastern Standard Time). Can you update the comment?

**Review by droazen — CHANGES_REQUESTED** [review]

File: .github/workflows/gatk-nightly-docker.yml (hunk 2 above)
Comment (jamesemery) [reply .github/workflows/gatk-nightly-docker.yml:4]: good point. I don't know how time zones work

File: .github/workflows/gatk-nightly-docker.yml [hunk 3]
```diff
@@ -1,10 +1,9 @@
 name: Build and Upload GATK Nightly Docker Image
 on:
   schedule:
-    # trigger the job the image at 1am est every night (UTC 20:00)
-    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
-    - cron: '0 20 * * *'
+    - cron: '17 05 * * *' # trigger the job the image at 12:17am EST every night (UTC 05:17)
```
Comment (droazen) [inline .github/workflows/gatk-nightly-docker.yml:4]: "the job the image" -> "the job to build the image"

**Review by droazen — APPROVED** [review]
One typo, then this can be merged. You should wait until well after 4 PM, however, to test whether the build scheduled for today happens -- recommend waiting until 5 PM.

**Pull request merged**